olive/engine/packaging/packaging_generator.py

```diff
@@ -565,7 +572,42 @@ def _generate_onnx_mlflow_model(model_dir: Path, inference_config: Dict):
     return mlflow_model_path
 
 
-def _package_onnxruntime_packages(tempdir: Path, pf_footprint: "Footprint"):
+def _package_onnxruntime_genai_runtime_dependencies(tempdir: Path):
+    # pylint: disable=not-an-iterable
+    installed_packages = [
+        pkg
+        for pkg in pkg_resources.working_set
+        if pkg.key.startswith("onnxruntime-genai") or pkg.project_name.startswith("onnxruntime-genai")
+    ]
+    if not installed_packages:
+        logger.warning("ONNXRuntime-GenAI package is not installed. Skip packaging runtime packages.")
+        return
+
+    DOWNLOAD_COMMAND_TEMPLATE = Template(
+        f"{sys.executable} -m pip download $package_name==$version --no-deps -d $python_download_path"
+    )
+    python_download_path = tempdir / "ONNXRuntimePackages" / "python"
+    python_download_path.mkdir(parents=True, exist_ok=True)
+    python_download_path = str(python_download_path)
+
+    for pkg in installed_packages:
+        pkg_name = pkg.key if pkg.key.startswith("onnxruntime-genai") else pkg.project_name
+        download_command = DOWNLOAD_COMMAND_TEMPLATE.substitute(
+            package_name=pkg_name, version=pkg.version, python_download_path=python_download_path
+        )
+
+        try:
+            run_subprocess(download_command)
+        except Exception:
+            logger.exception(
+                "Failed to download %s package. Manually download & install the required package.", pkg_name
```

Comment:
Nit: `Please manually download & install the required package`, otherwise it doesn't sound like an action to the user lol

Reply:
Done!